packages/theme/src/index.ts

```diff
@@ -114,6 +117,20 @@ const extension: JupyterFrontEndPlugin<ILauncher> = {
       palette.addItem({ command: CommandIDs.create, category: 'Launcher' });
     }
 
+    commands.addCommand(CommandIDs.openHelp, {
+      label: 'Documentation',
+      icon: helpIcon,
+      execute: (args: any) => {
+        window.open('https://elyra.readthedocs.io/en/latest/', '_blank');
```

Comment:
Do we want this to always open latest? Some sort of versioning or using stable might be better?

Reply:
Master -> latest

Reply:
yea, but then when a user installs Elyra 1.X in a couple months it would open 2.X documentation

Reply:
Yes, the release process need to handle this

Reply:
I have updated the release checklist to take care of this
https://github.com/elyra-ai/elyra/issues/503